common/chain/pubdata_helper.go

```diff
@@ -28,7 +28,7 @@ import (
 	"github.com/bnb-chain/zkbnb/types"
 )
 
-func ParseRegisterZnsPubData(pubData []byte) (tx *types.RegisterZnsTxInfo, err error) {
+func ParseRegisterZnsPubData(pubData []byte) (tx *legendTxTypes.RegisterZnsTxInfo, err error) {
```

Comment:
Do we need to rename `legendTxTypes` to other name? `legend` word is wired now.

Reply:
Agree